File: 01_ProgrammingBasics/_05_LoopsPartTwoMoreExercises/_05_AverageNumber.java
package _01_ProgrammingBasics._05_LoopsPartTwoMoreExercises;

import java.util.Scanner;

public class _05_AverageNumber {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Add variable
        double sum = 0;

        for (int i = 0; i < n; i++) {
            double number = Double.parseDouble(scan.nextLine());
            sum += number;
        }

        // Print the average
        System.out.printf("%.2f", sum/n);
    }
}

File: 23_C#Basics/_05_LoopsPartTwoMoreExercises/_05_AverageNumber/_05_AverageNumber.cs
using System;

public class _05_AverageNumber
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        // Add variable
        double sum = 0;

        for (int i = 0; i < n; i++)
        {
            double number = double.Parse(Console.ReadLine());
            sum += number;
        }

        // Print the average
        Console.WriteLine($"{sum / n:F2}");
    }
}
